osmosis-1/osmosis-1.assetlist.json

```diff
@@ -562,6 +562,35 @@
         "svg": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/btsg.svg"
       },
       "coingecko_id": "bitsong"
+    },
+    {
+      "description": "Panacea is a blockchain which is the key infrastructure for our services to reinvent the healthcare experience",
```

Comment:
```suggestion
      "description": "Panacea is a public blockchain launched by MediBloc, which is the key infrastructure for reinventing the patient-centered healthcare data ecosystem",
```

Reply:
Let's not use the term `our service` because Panacea is for everyone who wants to participate in the patient-centered healthcare data ecosystem.

Reply:
OK!